src/main/java/sirius/biz/jdbc/DatabaseController.java

```diff
@@ -89,33 +89,39 @@ public void sql(WebContext ctx) {
     public void executeQuery(WebContext ctx, JSONStructuredOutput out) throws SQLException {
         Watch w = Watch.start();
 
-        String database = ctx.get("db").asString(defaultDatabase);
-        Database db = determineDatabase(database);
-        String sqlStatement = ctx.get("query").asString();
-        SQLQuery qry = db.createQuery(sqlStatement);
-
-        if (isDDSStatement(sqlStatement)) {
-            // To prevent accidential damage, we try to filter DDS queries (modifying the database structure) and
-            // only permit them against our system database.
-            if (!Strings.areEqual(database, defaultDatabase)) {
-                throw Exceptions.createHandled()
-                                .withSystemErrorMessage(
-                                        "Cannot execute a DDS query against this database. This can be only done for '%s'",
-                                        database)
-                                .handle();
-            }
-
-            out.property("rowModified", qry.executeUpdate());
-        } else if (isModifyStatement(sqlStatement)) {
-            out.property("rowModified", qry.executeUpdate());
-        } else {
-            Monoflop monoflop = Monoflop.create();
-            qry.iterateAll(r -> outputRow(out, monoflop, r), new Limit(0, ctx.get("limit").asInt(DEFAULT_LIMIT)));
-            if (monoflop.successiveCall()) {
-                out.endArray();
+        try {
+            String database = ctx.get("db").asString(defaultDatabase);
+            Database db = determineDatabase(database);
+            String sqlStatement = ctx.get("query").asString();
+            SQLQuery qry = db.createQuery(sqlStatement);
+
+            if (isDDSStatement(sqlStatement)) {
+                // To prevent accidential damage, we try to filter DDS queries (modifying the database structure) and
```

Comment:
What does DDS means? I know the term DDL (Data Definition Language).

Reply:
DDS --> DDL?